cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -1200,14 +1249,18 @@ func (s *Supervisor) composeMergedConfig(config *protobufs.AgentRemoteConfig) (c
 	return configChanged, nil
 }
 
-func (s *Supervisor) handleRestartCommand() error {
+func (s *Supervisor) handleRestartCommand(ctx context.Context) error {
+	_, span := s.getTracer().Start(ctx, "HandleRestartCommand")
```

Comment:
```suggestion
	_, span := s.getTracer().Start(ctx, "handleRestartCommand")
```
not sure if you meant to use the exact function name?

Reply:
here i was not sure if i should consistently start with upper case for the span names, or use the exact function name - however the latter option might make more sense, so I will double check and make sure to use the exact function names